crates/swc/Cargo.toml

```diff
@@ -27,8 +27,11 @@ concurrent = [
 ]
 debug = ["swc_ecma_visit/debug"]
 node = ["napi", "napi-derive"]
-plugin = [
-  "swc_plugin_runner",
+plugin_embedded_runtime = [
```

Comment:
Why is this required?

I think user of the `swc` crate can depend on `swc_plugin_runner/embedded_runtime` or `swc_plugin_runner/native_runtime` instead?

Reply:
This is probably I'm making some mistakes if you felt bit odd. My thought was each package have deps to `swc` can select features without transitive deps feature selection, also swc itself uses own feature selection like https://github.com/swc-project/swc/pull/4012/files#diff-8987cc214ab0c7d63b833d31785ecb48d89d1bcc204d42508c26ecd72becc7b9R72 .

Reply:
I think this is not required. I'll check using a fork

Reply:
I think https://github.com/swc-project/swc/pull/4027 will work, except `swc_cli` on platforms not supported by `cranelift`. We should exclude them via `--no-default-features`